Source/WebCore/css/CSSSelector.cpp

```diff
@@ -659,7 +660,7 @@ String CSSSelector::selectorText(StringView separator, StringView rightSide) con
         return previousSelector->selectorText(separator, builder);
     } else if (auto separatorText = separatorTextForNestingRelative(); !separatorText.isNull()) {
         // We have a separator but no tag history which can happen with implicit relative nesting selector
-        return separatorText + builder.toString();
+        return makeString(separatorText, builder.toString());
```

Comment:
Should we teach `makeString` to work directly with a `StringBuilder` argument without creating an additional intermediate `String`? Should we add `StringBuilder::prepend` and can it be efficient enough to be OK?